plugins/modules/virtual_disk.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2023, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+module: virtual_disk
+
+author:
+  - Domen Dobnikar (@domen_dobnikar)
+short_description: Handles virtual disks on Hypercore cluster.
+description:
+  - Can create or delete virtual disk on cluster.
+  - Creates virtual disk from local disk file.
+version_added: 1.2.0
+extends_documentation_fragment:
+  - scale_computing.hypercore.cluster_instance
+seealso:
+  - module: scale_computing.hypercore.virtual_disk_info
+options:
+  file_location:
+    type: str
+    description: Location of the disk file.
+  file_name:
```

Comment:
Is name/source enough information for user to understand they need to specify a file instead of something else?
But from consistency POV I agree we should have the same parameter names where possible.